Packs/Prometheus/Integrations/Prometheus/Prometheus.yml

```diff
@@ -0,0 +1,128 @@
+category: Database
+commonfields:
+  id: Prometheus
+  version: -1
+name: Prometheus
+display: Prometheus
```

Comment:
please add sections to your yml file to specify the type of param, you can find an example of it in the repo if you search the word section

Reply:
@merit-maita Thanks for the recommendations, did I get the section tags right here? [033c29a](https://github.com/demisto/content/pull/41209/commits/033c29a1baa399cfd97a892793412dc53cacfc51)